### PR TITLE
Fix build by providing test property values on required properties an…

### PR DESCRIPTION
**Pull request by wtritch**
…d default values on optional properties.

### CODE REVIEW

File: webapp/src/main/resources/application.yml [hunk 1]
```diff
@@ -43,8 +43,8 @@ cloud:
   aws:
     credentials:
       #AWS credentials are necessary if reading assessment files from S3
-      accessKey: ${rdw.aws.access.key}
-      secretKey: ${rdw.aws.secret.key}
+      accessKey: ${rdw.aws.access.key:""}
+      secretKey: ${rdw.aws.secret.key:""}
```
Comment (wtritch) [inline webapp/src/main/resources/application.yml]: If not using S3 for item information, I believe these can be null/empty.  @malaffoon?

File: webapp/src/test/resources/application-test.yml [hunk 2]
```diff
@@ -9,4 +9,9 @@ permissionservice:
...
 
 application:
-  state: CA
\ No newline at end of file
+  state: CA
+
+app:
+  iris:
+    vendorId: vendorId
+    url: http://my-iris.somewhere.org/iris/
```
Comment (wtritch) [inline webapp/src/test/resources/application-test.yml]: I added test-only values when a property is required.

File: webapp/src/main/resources/application.yml (hunk 1 above)
Comment (bgsavage) [inline webapp/src/main/resources/application.yml]: Yes it should serve up your local item xmls then.

File: webapp/src/test/resources/application-test.yml (hunk 2 above)
Comment (bgsavage) [inline webapp/src/test/resources/application-test.yml]: It will only "fail fast" if you navigate to the right section in the UI.

**Pull request merged**